app/src/userpages/i18n/en.po

```diff
@@ -1125,7 +1125,7 @@ msgid "modal.shareResource.linkCopied"
 msgstr "Link copied"
 
 msgid "modal.shareResource.warnUnsavedChanges"
-msgstr "To update your permissions please save your changes. To discard them, click cancel."
+msgstr "To update your permissions please **save** your changes. To discard them, click **cancel**."
```

Comment:
We can render markdown stuff now. Example:
```jsx
<Md inline>
    {I18n.t('modal.shareResource.warnUnsavedChanges')}
</Md>
```
gives
```html
To update your permissions please <strong>save</strong> your
changes. To discard them, click <strong>cancel</strong>.
```
`inline` flag makes it skip `<p>` around the output cause only certain "inline" tags are allowed in that scenario, FYI.